### PR TITLE
26064 issuefix

### PR DESCRIPTION
**Pull request by divyajyothi5321**
### Fixed Issues (if relevant)

1. magento/magento2#26064: Incorrect Error Message While Sharing Wish list more than Specified Email Address Value in Admin Configuration

### Manual testing scenarios (*)

1. Go to frontend and login as Cutomer.
2. Add any product to wish list.
3. Go to My account dashboard -> My Wish list
4. Enter around 20 to 30 email addresses and share wish list.
5. The issue is fixed with this code. Please refer the screenshot.

wishlist-expected

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by divyajyothi5321** [comment]
@engcom-Bravo the static check is always failing at the part of the code which we haven't touched, please let us know what action we need to take here.

**Review by krzksz — CHANGES_REQUESTED** [review]
Hello @divyajyothi5321, thank you for your contribution! Can you please adjust the rest of the code so that static tests pass?

**Comment by divyajyothi5321** [comment]
@krzksz I have made text change for the message and it is properly formatted. Can you Please let me know the exact place where I need to make the change to pass the static test.

File: app/code/Magento/Wishlist/Controller/Index/Send.php [hunk 1]
```diff
@@ -204,7 +204,7 @@ public function execute()
                 $error = __('Please enter an email address.');
             } else {
                 if (count($emails) > $emailsLeft) {
-                    $error = __('This wish list can be shared %1 more times.', $emailsLeft);
+                    $error = __('Maximum of %1 Emails can be Sent.', $emailsLeft);
```
Comment (krzksz) [inline app/code/Magento/Wishlist/Controller/Index/Send.php]: Please adjust the wording to `Maximum of %1 emails can be sent.` (letter casing).

**Review by krzksz — CHANGES_REQUESTED** [review]
Static tests ask you to remove `@package Magento\Wishlist\Controller\Index` and add some meaningful description to the class.

**Comment by divyajyothi5321** [comment]
@krzksz even after changing the file as per your suggestion, the static test failed once again. could you please review and let us know the fix.

**Review by krzksz — CHANGES_REQUESTED** [review]
There is one error displayed:
```
PHPStan detected violation(s):
 ------ -------------------------------------------------------------------- 
  Line   app/code/Magento/Wishlist/Controller/Index/Send.php                 
 ------ -------------------------------------------------------------------- 
  325    Method Magento\Wishlist\Controller\Index\Send::getRssLink() should  
         return mixed but return statement is missing.                       
 ------ -------------------------------------------------------------------- 

 [ERROR] Found 1 error 
```
FYI: you can check the error yourself by clicking the **Details** link next to the certain test.

File: app/code/Magento/Wishlist/Controller/Index/Send.php (hunk 1 above)
Comment (krzksz) [reply app/code/Magento/Wishlist/Controller/Index/Send.php]: This still should be adjusted.

**Review by krzksz — CHANGES_REQUESTED** [review]

**Review by krzksz — APPROVED** [review]

**Comment by krzksz** [comment]
Hey @divyajyothi5321, thank you for the contribution!
According to our Definition of Done, all code changes should be covered by tests.

**Comment by magento-engcom-team** [comment]
Hi @krzksz, thank you for the review.
[ENGCOM-6518](https://jira.corp.magento.com/browse/ENGCOM-6518) has been created to process this Pull Request

**Comment by divyajyothi5321** [comment]
@krzksz how to cover auto testing kindly can you please let me know

**Comment by engcom-Golf** [comment]
HI @divyajyothi5321 @krzksz all changes covered by unit test, could you please review ?

**Review by krzksz — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @krzksz, thank you for the review.
[ENGCOM-6518](https://jira.corp.magento.com/browse/ENGCOM-6518) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Comment by engcom-Golf** [comment]
Functional test fails on 2.4-develop branch, fix in -> https://github.com/magento/magento2/pull/26225

**Pull request merged**